mix.exs

```diff
@@ -1,6 +1,6 @@
 # Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
 
-antikythera_dep = {:antikythera, [git: "git@github.com:access-company/antikythera.git", ref: "f82ba97b1b2cfde87e85308d76766f84a34f8984"]}
+antikythera_dep = {:antikythera, [git: "https://github.com/access-company/antikythera.git", ref: "f82ba97b1b2cfde87e85308d76766f84a34f8984"]}
```

Comment:
Here we have been using the SSH protocol just for a historical reason (in order to authenticate access to the private repo).
Could you use the `:github` as a shortcut: `github: "access-company/antikythera"`?
Then I'll merge this in.

Reply:
Sure!